frontend/src/app/login/page.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+const LoginPage = () => {
+  return (
+    <section className="h-screen mx-auto flex flex-col items-center  justify-center">
+      <div className="bg-white/10 p-4 rounded-xl">
+        <p className="text-center text-3xl">Acessar conta</p>
+        <form action="" className="flex flex-col space-y-2">
+          <div className="flex flex-col w-full min-w-xs">
+            <label htmlFor="" className="text-lg font-light">
+              Email
+            </label>
+            <input
+              type="email"
+              className="bg-light w-full py-2 text-lg text-dark pl-2 rounded-md"
+            />
+          </div>
```

Comment:
```suggestion
          <div className="flex flex-col w-full min-w-xs">
            <label htmlFor="" className="text-lg font-light">
              Email
            </label>
            <input
              type="email"
              className="bg-light w-full py-2 text-lg text-dark pl-2 rounded-md"
            />
          </div>
```

Essa parte do código deverá virar um componente.